lib/src/integrationTest/java/io/github/taz03/jia/friendships/FriendshipsTest.java

```diff
@@ -0,0 +1,28 @@
+package io.github.taz03.jia.friendships;
+
+import static io.github.taz03.jia.TestProxy.client;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.concurrent.ExecutionException;
+import io.github.taz03.jia.requests.friendships.FriendshipsFeedRequest;
+import io.github.taz03.jia.responses.friendships.FriendshipsFeedResponse;
+import org.junit.jupiter.api.Test;
+
+public class FriendshipsTest {
+
+    @Test
+    public void friendshipsFeedFollowingTest() throws ExecutionException, InterruptedException {
```

Comment:
we don't care about the exact exception so just do `throws Exception`

Reply:
fixed